MarvelApiClient/build.gradle

```diff
@@ -10,10 +10,12 @@ repositories {
 dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.11'
     testCompile 'org.mockito:mockito-all:1.9.5'
+    testCompile 'com.squareup.okhttp:mockwebserver:2.5.0'
+    testCompile 'commons-io:commons-io:2.4'
 
     compile 'com.squareup.okhttp:okhttp:2.5.0'
     compile 'com.squareup.retrofit:retrofit:2.0.0-beta2'
     compile 'com.squareup.retrofit:converter-gson:2.0.0-beta2'
-    compile 'com.squareup.okhttp:logging-interceptor:2.6.0-SNAPSHOT'
+    compile 'com.squareup.okhttp:logging-interceptor:2.7.0-SNAPSHOT'
```

Comment:
because 2.6 was not on live when I wrote this code, now it's on live, good catched, i going to change it.